starter-code/iteration3/iteration3/src/app/app.component.html

```diff
@@ -0,0 +1,13 @@
+<ul class="flex">
+  <li *ngFor="let oneChampionship of championship">
+    <h2>{{oneChampionship.year}}</h2>
+    <h3>{{oneChampionship.name}}</h3>
+    <img src={{oneChampionship.championUrl}}>
+    <h4>Best Players</h4>
+    <ul>
+      <li *ngFor="let oneList of list">
+        <p>{{oneList.list}}</p>
```

Comment:
And here you don't need to put .list as you are already iterating inside this :)

So you only need to refer to "one" (if we stick t the line I write before) and it will do the tricks